flexget/tests/test_exists_movie.py

```diff
@@ -120,6 +130,19 @@ def test_existing_files(self, execute_task):
             'accepted', title='Gone.Missing.2013'
         ), 'Gone.Missing.2013 should have been accepted'
 
+    def test_same_name_diff_year(self, execute_task):
```

Comment:
nice test. Maybe add another one as a negative test, where exist movie rejects a movie with the same year?

Reply:
I already added that expect https://github.com/Flexget/Flexget/pull/2455/files#diff-e1ead679e7478a7d54927d28c3a18961R136